File: basketservice/menu_item.py
from pydantic import BaseModel, Field
from typing import Optional
from uuid import uuid4, UUID


class MenuItem(BaseModel):
    customer_id: Optional[UUID] = Field(default=None, description="The ID of the user associated with the menu item")
    menu_item_guid: UUID = Field(default_factory=uuid4, description="Unique identifier for the menu item")
    name: str
    menu_type: str
    description: Optional[str] = None
    price: float

File: orderservice/Models/MenuItem.cs
using System.ComponentModel.DataAnnotations;

public class MenuItem
{
    [Key]
    public Guid MenuItemGuid { get; set; }

    [Required]
    public Guid OrderGuid { get; set; }

    [Required]
    public Order Order { get; set; } = null!;

    [Required]
    public required string Name { get; set; }

    [Required]
    public required string MenuType { get; set; }

    [Required]
    public required string Description { get; set; }

    [Required]
    public float Price { get; set; }
}